server/src/main/scala/org/scassandra/server/priming/cors/CorsSupport.scala

```diff
@@ -0,0 +1,43 @@
+package org.scassandra.server.priming.cors
```

Comment:
Ah ok, sure will add it shortly. No problem.

> On 31 May 2016, at 13:21, Andrew Tolbert <notifications@github.com> wrote:
> 
> In server/src/main/scala/org/scassandra/server/priming/cors/CorsSupport.scala <https://github.com/scassandra/scassandra-server/pull/162#discussion_r65170832>:
> 
> > @@ -0,0 +1,43 @@
> > +package org.scassandra.server.priming.cors
> For any new file would be good to add the copyright header
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/scassandra/scassandra-server/pull/162/files/25f99748f09f8a6290c5b72e805bde6764026c8e#r65170832>, or mute the thread <https://github.com/notifications/unsubscribe/AEcH5zdhYXb1Zmmsc3aEDyU6e13KT8f6ks5qHCfegaJpZM4Io98s>.
>